cmd/immuadmin/command/user.go

```diff
@@ -143,14 +145,14 @@ func (cl *commandline) changeUserPassword(username string, oldpassword []byte) (
 		return "Error Reading Password", nil
 	}
 	if err = auth.IsStrongPassword(string(newpass)); err != nil {
-		return "Password does not meet the requirements. It must contain upper and lower case letter, digits, numbers, puntcuation mark or symbol.", nil
+		return "", fmt.Errorf("Password does not meet the requirements. It must contain upper and lower case letter, digits, numbers, puntcuation mark or symbol.")
```

Comment:
I suppose it should be "letters"? Also, digits and numbers seem redundant, don't they? "puntcuation" is a typo.